### PR TITLE
80 new copy

### LINKED ISSUE #80: Rework Copy to reflect project better
## Changes
Copy has an issue because the main explanatory paragraph lives below the fold on the landing page - as in main uses of the app we only want users to read that section if they're seeking info, it isn't part of their main journey. Also we're changing where we think we are with testing the app at present so some other sections are inaccurate

## Plan
- [x] rework info sections to duplicate that content.
- [x] rework that section to explain its intended function in a live app
- [x] rework info sections to reflect current project plan and introduce survey better
- [ ] update app with those reworkings
- [ ] test app with people who don't know the project

### PR DESCRIPTION
**Pull request by HedgehogHolly**
Fixes #80 

## Description

- Front page content updated
- Info button names updated
- Info content updated

### CODE REVIEW

File: src/Info.elm [hunk 1]
```diff
@@ -119,65 +119,61 @@ placeholderInfo =
...
+            [ """We include bite size information relating to a small number of fears and barriers that are likely to be stopping people from taking the next step. Keeping the
+            language simple we aim to be open and honest, with a reassuring tone. We resist the temptation to provide all the information people might need. Our goal is to offer just enough to
+            help people reach out to support services. Other websites have excellent longer content. """
             ]
       }
-    , { id = 2
+    , { id = 3
       , name = "Illustrations"
-      , slug = "illustrations"
+      , slug = "Illustrations"
```
Comment (katjam) [inline src/Info.elm]: keep the little i - urls are better all lowercase and this turns up in the url (I think)

**Review by katjam — CHANGES_REQUESTED** [review]
Looks great apart from the 'I' in illustrations slug

**Comment by HedgehogHolly** [comment]
Updated the slug @katjam

**Review by katjam — APPROVED** [review]
Perfect! :1st_place_medal:

**Pull request merged**